### PR TITLE
hiretrade Pledge

### PR DESCRIPTION
**Pull request by shivlim**

### CODE REVIEW

**Review by dckc — CHANGES_REQUESTED** [review]
Hi.  Thanks for your pledge! But we need you to move it down under a folder rather than changing the top level README.md

**Comment by shivlim** [comment]
> Hi. Thanks for your pledge! But we need you to move it down under a folder rather than changing the top level README.md

Updated now. Thanks

File: hiretrade/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+I, hiretrade hereby commit to the following work and support for the benefit of the Agoric Network and Community. If, for any reason, I do not meet these commitments then I understand that I may lose delegation staked to me.:
```
Comment (jeetraut) [inline hiretrade/README.md:1]:
```suggestion
[hiretrade Explorer Link](https://main.explorer.agoric.net/validator/agoricvaloper1s55crdh7fgn8z7vvhdfc52f8zwq03cng9wryuj)
I, hiretrade hereby commit to the following work and support for the benefit of the Agoric Network and Community. If, for any reason, I do not meet these commitments then I understand that I may lose delegation staked to me.:
```

**Review by jeetraut — CHANGES_REQUESTED** [review]
Add explorer Link

**Comment by shivlim** [comment]
> Add explorer Link

Done. Thanks

**Comment by jeetraut** [comment]
Hi. Thanks for the contribution, but we need your README.md file to be in a subfolder. Take a look at https://github.com/Agoric/validator-profiles/pull/2 for an example where the README.md file is BlockShark/README.md.

We could try to resolve the conflicts in this branch / PR, but that would be kind of a hassle, so... can we ask you to just make another PR?

**Pull request closed without merging**